### DISCUSSION TITLE
Configuring the Email Provider doesn't work as expected?

### DISCUSSION THREAD

**Discussion by divineforge** [strapi/strapi#7760] [Legacy Topics]
I'm trying to use the email provider - sendGrid , following the documentation instead of the default sendmail.

I notice the behaviour was it keep goes to 'sendMail' provider, (default one). I have tried to disable the plugin in the admin panel, it still go to the sendMail plugin.

To replicate:
- configure a sendGrid email provider, put inside a config/plugin.js (everything tested works fine, sendGrid can work)
- try to use Postman to call to 'forgot password' and it supposes to send an email, using the 'sendGrid' plugin that was installed, but it goes to the sendMail plugin (default one).

What I tried
- I put debugging lines in those modules to see which one it goes to, it seems always go to the sendMail plugin. 
- I also disabled the default email provider in the admin.

Questions:
- how do we switch email provider properly?

![Screenshot 2020-09-07 at 5 54 30 PM](https://user-images.githubusercontent.com/1146195/92375744-1e781100-f134-11ea-97a0-79b733bf739c.png)
<img width="1000" alt="Screenshot 2020-09-07 at 5 54 01 PM" src="https://user-images.githubusercontent.com/1146195/92375747-2041d480-f134-11ea-9230-224d7f77fb0f.png">

**Comment by derrickmehaffy** [comment]
Which version of Strapi are you on?

**Reply by divineforge** [reply]
version 3.1.4 thanks for checking. but I found the answer. silly me that it was caused by the filename. it has to be `plugins.js` instead of plugin.js (additional S needed in the name)
